src/index.js

```diff
@@ -65,6 +66,9 @@ chaiJestSnapshot.resetSnapshotRegistry = function resetSnapshotRegistry() {
   }
```

Comment:
Could you add the stuff you had in your beforeEach to `configureUsingMochaContext`?